### DISCUSSION TITLE
Moving workspace to a specific monitor output

### DISCUSSION THREAD

**Discussion by wreckdump** [i3/i3#4904] [Q&A]
I am trying to move a workspace to a specific monitor output in a normal terminal emulator not in a i3config file, and the solutions off of the reference manual keeps failing. For example, when I use the following command

`$ i3-msg workspace 2 output DisplayPort-2`

where the 'DisplayPort-2' is one of the monitors name from `xrandr` output, it just creates a workspace named '2 output DisplayPort-2'. Why is it doing this? Am I doing something wrong?

**Comment by RayZ0rr** [comment]
I think the command maybe

`i3-msg move workspace 2 to output DisplayPort-2`

**Reply by wreckdump** [reply]
Thanks you for the reply. Unfortunately, with your suggested command I just get a new workspace named '2 to output DisplayPort-2'

**Reply by RayZ0rr** [reply]
I see. In [this section](https://i3wm.org/docs/userguide.html#_moving_containers) they give an example like this:
```sh
# move the whole workspace to the next output
bindsym $mod+x move workspace to output right
```
So, I think you can go to the workspace you want to move and do: 
`i3-msg move workspace to output DisplayPort-2`

**Comment by orestisfl** [comment]
What you are trying to use is a config directive that can only go in the config file. There is no command that can do this.

**Reply by wreckdump** [reply]
Okay. What I am trying to do is that when I plug my laptop into my docking station, I want to move all the workspaces to different monitor outputs with a shell script (ideally). Is this possible with i3? Or are there any workaround for this?
